docs/_source/tutorials-examples/population-synthesis/10_binaries_pop_syn.ipynb

```diff
@@ -42,27 +37,36 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Creating the Initialisation File"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
+    "\n",
     "----\n",
+    "\n",
+    "## Initialisation File\n",
+    "\n",
     "To run population synthesis with POSYDON, a `population_params.ini` file is required.\n",
     "This file described how the stellar population is created and what prescriptions and parameters are implemented in specific steps.\n",
     "\n",
-    "POSYDON comes with a default `population_params_default.ini` file found at `PATH_TO_POSYDON/posydon/popsyn` or have a look [here](population_params.ini).\n",
+    "POSYDON comes with a default `population_params_default.ini` file found at `PATH_TO_POSYDON/posydon/popsyn` or have a look [here](../../../../../posydon/popsyn/population_params_default.ini).\n",
+    "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "    \n",
+    "   **Default values**\n",
+    "   \n",
+    "   The default values in the population_params.ini file included in POSYDON \n",
+    "   have not been calibrated, validated, or are used by the POSYDON team. They are\n",
+    "   often an ad-hoc choice, and the user should carefully consider the values\n",
+    "   of each parameter for their science case.\n",
```

Comment:
I'd suggest to add a link to the description of the parameters. We still need to be aware that others won't be willing to read all the docs before getting started.

Reply:
There's a link right below it.

Reply:
Sure, just be aware, all below isn't read yet and may be outside the screen.